rust-symcrypt/src/cipher/cbc.rs

```diff
@@ -0,0 +1,319 @@
+//! Functions for Cbc encryption and decryption. For more info please see symcrypt.h
+//!
+//! These functions are methods on the [AesExpandedKey] struct.
+//!
+//! # Examples
+//!
+//! ## AES-CBC Encryption in one go
+//!
+//! ```rust
+//! use symcrypt::cipher::AesExpandedKey;
+//! use hex::*;
+//!
+//! // Set up the key, chaining value, and plaintext
+//! let key = hex::decode("00000000000000000000000000000000").unwrap();
+//! let aes_cbc = AesExpandedKey::new(&key).unwrap();
+//! let mut chaining_value = hex::decode("00000000000000000000000000000000").unwrap();
+//! let mut plain_text = hex::decode("f34481ec3cc627bacd5dc3fb08f273e6").unwrap();
```

Comment:
>f34481ec3cc627bacd5dc3fb08f273e6 [](http://example.com/codeflow?start=38&length=32)

Something readable

Reply:
Gcm test do not show something readable, going to leave as is